client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -57,6 +57,11 @@ export default React.createClass( {
 
 	monthlyPrice: function() {
 		const { cost, currency } = this.props.cartItem;
+
+		if ( typeof cost === 'undefined' ) {
+			return this.translate( 'Loading price' );
```

Comment:
As long as you don't mind the height jump. This case should happen rarely enough, so I don't mind either way.